bin/downgrade

```diff
@@ -110,21 +110,14 @@ read_selection() {
   ((ans > 0 && ans <= $#)) && printf "%s" "${!ans}"
 }
 
-prompt_gettext() {
-  local key="$1" ans
-
-  eval_gettext "$key"
```

Comment:
As a quick update, I had to revert this simplification to `downgrade`'s call of `pacignore` because `xgettext` was not able to resolve this line and would assume that this prompt no longer requires localization support. IMO this is not a big loss because the reverted portion still looks compact and reads well IMO